internal/pkg/utils/deepcopy/deepcopy.go

```diff
@@ -148,14 +148,16 @@ func translateRecursive(clone, original reflect.Value, callback TranslateFunc, s
 
 	// If it is a map we create a new map and translate each value
 	case kind == reflect.Map:
-		clone.Set(reflect.MakeMap(originalType))
-		for _, key := range original.MapKeys() {
-			originalValue := original.MapIndex(key)
-			// New gives us a pointer, but again we want the value
-			cloneValue := reflect.New(originalValue.Type()).Elem()
-			steps := steps.Add(kind.String(), cast.ToString(key))
-			translateRecursive(cloneValue, originalValue, callback, steps, visited)
-			clone.SetMapIndex(key, cloneValue)
+		if !original.IsNil() {
```

Comment:
Bolo to nespravne v priklade, z ktoreho som cerpal:
https://gist.github.com/hvoecking/10772475#file-translate-go-L237-L245

`Slice` (vyssie v kode) sme opravili uz skor.